cmd/terraform-registry/main.go

```diff
@@ -35,6 +38,9 @@ var (
 	logLevelStr    string
 	logFormatStr   string
 
+	awsRegion   string
+	awsS3Bucket string
```

Comment:
prefix these vars with `s3`

Reply:
It might be cleaner to remove AWS from the names of these, as there are also multiple other providers supporting the S3 protocol by now. What do you think?

Reply:
Accepted:

That sounds great to me, there is currently a reliance on AWS credentials to build the buckets but nothing a feature request in the future couldn't address.

Reply:
Appreciate the review on this!